apps/site/assets/js/google-maps-helpers.js

```diff
@@ -38,7 +38,7 @@ const processAutocompleteResults = resolve => ({ predictions }) =>
 
 const predictionResults = predictions => ({
   locations: {
-    hits: predictions,
+    hits: predictions.map((p, i) => ({...p, id: `location-${i}`})),
```

Comment:
This is pretty janky, but the click handlers for the location results are added based on an `id`. This was previously determined using the `place_id`, but since we're no longer exposing that concept from the backend I've had to change the logic. Definitely open to other suggestions here, but this was the lowest-lift idea I came up with to hook up the click handlers properly.,